week3_ts/src/component/MainHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import { HandsomeGuy } from "@/assets/images";
+import styled from "styled-components";
+
+interface MainHeaderProps {
+    round: string;
+    matchWinners: React.MutableRefObject<HandsomeGuy[]>;
+    fighterList: HandsomeGuy[];
+}
+
+export default function MainHeader(props: MainHeaderProps) {
+    const { round, matchWinners, fighterList } = props;
+
+    // 현재 몇 라운드인지
+    const countRoundNum = (): number => {
+        return matchWinners.current.length + 1;
+    };
+    // 이번 강에는 총 몇 번의 라운드가 있는지
+    const totalRoundNum = (): number => {
+        return Math.ceil((fighterList.length + matchWinners.current.length * 2) / 2);
+    };
```

Comment:
단순 리턴 함수를 선언하신 이유가 무엇인가요??! 😲😲

```suggestion
    // 현재 몇 라운드인지
    const countRoundNum = matchWinners.current.length + 1;
    // 이번 강에는 총 몇 번의 라운드가 있는지
    const totalRoundNum = Math.ceil((fighterList.length + matchWinners.current.length * 2) / 2);
```

Reply:
그러게!! 나도 궁금하다!!!!!!!!!!

Reply:
화살표 함수를 사용해야한다는 생각을 계속 갖고 있어서 그런 거 같습니다 짚어주셔서 감자합니다